### PR TITLE
App config

### PR DESCRIPTION
**Pull request by smirnovaka**
footer view
@kgrgreer

### CODE REVIEW

File: src/foam/nanos/app/AppConfig.js [hunk 1]
```diff
@@ -19,13 +19,39 @@ foam.CLASS({
...
     },
     {
       class: 'String',
       name: 'url',
       value: 'http://localhost:8080/'
     },
+    {
+      class: 'String',
+      name: 'urlLabel',
+      value: 'Powered by Nanopay'
```
Comment (jlhughes) [inline src/foam/nanos/app/AppConfig.js]: Believe this should be nanopay with lowercase 'n'.

File: src/foam/nanos/u2/navigation/FooterView.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      float: left;
+    }
+  `,
+
+  properties: [
+    {
+      class: 'String',
+      name: 'aboutLabel',
+      factory: function () {
+        return 'Powered by nanopay';
```
Comment (jlhughes) [inline src/foam/nanos/u2/navigation/FooterView.js]: Should this be using the urlLabel from appConfig?

File: src/foam/nanos/app/AppConfig.js [hunk 3]
```diff
@@ -19,13 +19,39 @@ foam.CLASS({
...
+    },
+    {
+      class:'String',
+      name: 'termsAndCondLabel',
+      value: 'Terms and Conditions'
+    },
+    {
+      class: 'String',
+      name: 'termsAndCondLink',
+      value: 'net.nanopay.ui.modal.TandCModal'
```
Comment (kirkeaton) [inline src/foam/nanos/app/AppConfig.js]: You can't use this class here. It breaks the standalone version of foam2

File: src/foam/nanos/app/AppConfig.js (hunk 1 above)
Comment (eatonk) [reply src/foam/nanos/app/AppConfig.js]: This and any other reference to nanopay explicitly should be removed and set elsewhere in our repositories

**Review by kirkeaton — CHANGES_REQUESTED** [review]

File: src/foam/nanos/app/AppConfig.js (hunk 1 above)
Comment (kirkeaton) [reply src/foam/nanos/app/AppConfig.js]: This and any other references to nanopay should be removed from this repository and stored in one of our repos

Comment (kgrgreer) [reply src/foam/nanos/app/AppConfig.js]: "FOAM Powered" might be a suitable default.

File: src/foam/nanos/u2/navigation/FooterView.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+          .start(this.GO_TO,{ label$: this.appConfig.urlLabel$ }).end()
+          .add('|')
+          .start(this.GO_TO_TERM, { label$: this.appConfig.termsAndCondLabel$ }).end()
+          .add('|')
+          .start(this.GO_TO_PRIVACY, { label$: this.appConfig.privacy$ }).end()
+          .add('|')
+          .start().addClass('mode').add(this.appConfig.mode$.map(function(m) { return m.label; }), ' version: ', this.appConfig.version$).end()
+        .end()
+        .start('div').addClass('col').addClass('copyright-label')
+         // .start('p').add('Copyright © 2018 ' + this.webApp + '. All rights reserved.').end()
```
Comment (kirkeaton) [inline src/foam/nanos/u2/navigation/FooterView.js]: Remove this commented out code

File: src/foam/nanos/app/AppConfig.js [hunk 5]
```diff
@@ -17,14 +17,36 @@ foam.CLASS({
...
     {
       class: 'String',
-      name: 'url',
-      value: 'http://localhost:8080/'
+      name: 'url'
+    },
+    {
+      class: 'String',
+      name: 'urlLabel',
+      value: 'FOAM powered'
```
Comment (kgrgreer) [inline src/foam/nanos/app/AppConfig.js]: FOAM Powered

**Review by kgrgreer — APPROVED** [review]

**Pull request merged**